File: app/src/main/java/com/a60n1/ejashkojme/models/Conversation.java
package com.a60n1.ejashkojme.models;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Conversation {

    public boolean seen;
    public long timestamp;

    public Conversation() {

    }

    public Conversation(boolean seen, long timestamp) {
        this.seen = seen;
        this.timestamp = timestamp;
    }
}

File: app/src/main/java/com/a60n1/ejashkojme/models/Conversation.kt
package com.a60n1.ejashkojme.models

import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
class Conversation {
    @JvmField
    var seen = false
    var timestamp: Long = 0

    @Suppress("unused")
    constructor()

    @Suppress("unused")
    constructor(seen: Boolean, timestamp: Long) {
        this.seen = seen
        this.timestamp = timestamp
    }
}